C#/Libraries/SkillTools/README.md

```diff
@@ -3,6 +3,17 @@
 The former MistyRobotics.Tools nuget package, which is being deprecated.
 Contains helper functionality for http requests, simple data stores, and asset uploading and use.
 
-Copyright 2020 Misty Robotics
-Licensed under the Apache License, Version 2.0
-http://www.apache.org/licenses/LICENSE-2.0
\ No newline at end of file
+---
+
+**WARRANTY DISCLAIMER.**
+
+* General. TO THE MAXIMUM EXTENT PERMITTED BY APPLICABLE LAW, MISTY ROBOTICS PROVIDES THIS SAMPLE SOFTWARE “AS-IS” AND DISCLAIMS ALL WARRANTIES AND CONDITIONS, WHETHER EXPRESS, IMPLIED, OR STATUTORY, INCLUDING THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE, TITLE, QUIET ENJOYMENT, ACCURACY, AND NON-INFRINGEMENT OF THIRD-PARTY RIGHTS. MISTY ROBOTICS DOES NOT GUARANTEE ANY SPECIFIC RESULTS FROM THE USE OF THIS SAMPLE SOFTWARE. MISTY ROBOTICS MAKES NO WARRANTY THAT THIS SAMPLE SOFTWARE WILL BE UNINTERRUPTED, FREE OF VIRUSES OR OTHER HARMFUL CODE, TIMELY, SECURE, OR ERROR-FREE.
+* Use at Your Own Risk. YOU USE THIS SAMPLE SOFTWARE AND THE PRODUCT AT YOUR OWN DISCRETION AND RISK. YOU WILL BE SOLELY RESPONSIBLE FOR (AND MISTY ROBOTICS DISCLAIMS) ANY AND ALL LOSS, LIABILITY, OR DAMAGES, INCLUDING TO ANY HOME, PERSONAL ITEMS, PRODUCT, OTHER PERIPHERALS CONNECTED TO THE PRODUCT, COMPUTER, AND MOBILE DEVICE, RESULTING FROM YOUR USE OF THIS SAMPLE SOFTWARE OR PRODUCT.
+
+Please refer to the Misty Robotics End User License Agreement for further information and full details: https://www.mistyrobotics.com/legal/end-user-license-agreement/
+
+--- 
+
+*Copyright 2020 Misty Robotics*<br>
```

Comment:
Should these <br>'s be in the README.md files? They disappear when I view the file in this PR tool, but when I open them with NOTEPAD++ or Chrome on my PC I see the text "<br>"

Reply:
Do you mean the disclaimer copy in general? Yes, that should be in the README.md files  👍